tempesta_fw/t/unit/test_http_sticky.c

```diff
@@ -149,8 +148,7 @@ tfw_connection_send(TfwConnection *conn, TfwMsg *msg,
 
 	mock.tfw_connection_send_was_called += 1;
 
-	skb = ss_skb_peek(&msg->skb_list);
-	while (skb) {
+	while ((skb = ss_skb_peek(&msg->skb_list))) {
```

Comment:
Endless loop? `ss_skb_peek()` always return first skb in the list, so following `ss_skb_next()` seems should be replaced by `ss_skb_unlink()`.